homeassistant/components/stream/__init__.py

```diff
@@ -38,12 +52,18 @@
 )
 
 
-@bind_hass
-def request_stream(hass, stream_source, *, fmt="hls", keepalive=False, options=None):
-    """Set up stream with token."""
+def create_stream(hass, stream_id, stream_source, options=None):
```

Comment:
Would it make sense to just generate a stream_id inside this method? 

Or actually, do we need a stream ID ? The objects are now managed by `camera`, so the camera can deal with it (it stores it on CameraEntity instance now)

Reply:
We don't need it if we drop the record service (I had a similar thought above that it isn't needed so I'm on board with this)